rules/manager.go

```diff
@@ -264,16 +264,21 @@ type Group struct {
 	logger log.Logger
 
 	metrics *Metrics
+
+	ruleGroupPostProcessFunc RuleGroupPostProcessFuncType
 }
 
+type RuleGroupPostProcessFuncType func(g *Group, log log.Logger) error
```

Comment:
This type should be clearly documented, especially:

- When is the function called, what is the purpose of the function.

Reply:
Done